Api/OrderReceivedUrlInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2019 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Api;
+
+/**
+ * Create Order interface.
+ * @api
+ */
+interface OrderReceivedUrlInterface
+{
+    /**
+     * Create order.
```

Comment:
I believe this is more like "Display order confirmation screen"

Reply:
updated.